LeadDuplicator.js

```diff
@@ -1,17 +1,15 @@
-let duplicatesCount = 5;
-let formData = ZDK.Page.getForm().getValues(); // collecting the data from form
+const duplicatesCount = 5;
+const formData = ZDK.Page.getForm().getValues(); // collecting the data from form
+const leadObjects = [];
 
-let tempLead = new ZDK.Apps.CRM.Leads.Models.Leads();
-
-// shallow copy the form data to the tempLead
-for (key in formData) {
-    log("Storing : " + key + " " + formData[key]);
-    tempLead[key] = formData[key];
+for (let i = 0; i < duplicatesCount; i++) {
+  const tempLead = new ZDK.Apps.CRM.Leads.Models.Leads();
```

Comment:
Instead of creating a tempLead object for every duplicate and assigning the formData each time, can we use a single tempLead object and use it for all the duplicates ??

In that way I thinl we can reduce the redundant assigning operations and improve the performance...

Reply:
We are working on object reference so, it would overwrite old values.